pkg/process/config/helpers.go

```diff
@@ -0,0 +1,15 @@
+package config
+
+import (
+	"os"
+)
+
+// GetSocketPath exports the socket path we are using for the system probe
+func GetSocketPath() string {
```

Comment:
Is it possible to call  `proc_config.loadEnvVariables` instead of adding this one off function for getting DD_SYSPROBE_SOCKET to work ?

Reply:
I was hesitant to do that because I think it would be overwriting the global Datadog config object for the entire main agent binary.

Reply:
And that function will overwrite things like the API key, etc